csr/csr_test.go

```diff
@@ -189,7 +189,16 @@ func TestNamesFromCSR(t *testing.T) {
 			[]string{"a.com", "b.com"},
 		},
 		{
-			"no explicit CN, too long leading SANs",
+			"no explicit CN, all SANs too long to be CNs",
```

Comment:
nit: There's only ever one CN, so I'd say "...too long to be the CN"

Same on lines 201 and 212, similar on lines 225 and 233